app/controllers/providers/check_provider_answers_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Providers
+  class CheckProviderAnswersController < BaseController
+    include Providers::ApplicationDependable
+
+    def index
+      @proceeding = @legal_aid_application.proceeding_types[0]
+      @applicant = @legal_aid_application.applicant
+      @address = html_address(@applicant.addresses[0])
+    end
+
+    def confirm
+      legal_aid_application.provider_submit!
+      CitizenEmailService.new(legal_aid_application).send_email
+    end
+
+    private
+
+    def html_address(address)
+      [address.organisation, address.address_line_one, address.address_line_two, address.city, address.postcode].compact.reject(&:blank?).join('</br>').html_safe
+    end
```

Comment:
Two things ...

First, this method is very similar to `address#full_address`. I'd suggest modifying that method and using it here. Something like:
```ruby
class Address < ApplicationRecord
  ...
  def full_address(joined_by: ', ')
    [organisation, address_line_one, address_line_two, city, postcode].compact.reject(&:blank?).join(joined_by)
  end
```
Then here you can do:
```
  address.full_address(joined_by: '</br>')
```

Second you must not use `.html_safe` here as address is user input. So it is not safe to blanket sanitize it like this. I expect the thing you are trying to sanitize is `</br>` so just sanitize that:

```ruby
sanitize address.full_address(joined_by: '</br>'), tags: ['br']
```
As you are doing this in the controller, you'll probably need to `include ActionView::Helpers::SanitizeHelper`. Alternatively and probably better:

1. Assign the address like this: `@address = @applicant.addresses.first`
2. Then use a helper method to display the address as needed (`sanitize` will be available within a helper).